### ISSUE TITLE
Remove non-interface/mixin data from api directory

### ISSUE THREAD

**Issue by saschanaz** [mdn/browser-compat-data#6810]
Found some non-interface things:

Dictionaries:

* [`ConstrainBoolean`](https://github.com/mdn/browser-compat-data/tree/master/api/ConstrainBoolean.json)
* [`ConstrainDOMString`](https://github.com/mdn/browser-compat-data/tree/master/api/ConstrainDOMString.json)
* [`ConstrainDouble`](https://github.com/mdn/browser-compat-data/tree/master/api/ConstrainDouble.json)
* [`ConstrainLong`](https://github.com/mdn/browser-compat-data/tree/master/api/ConstrainLong.json)
* [`ConstrainULong`](https://github.com/mdn/browser-compat-data/tree/master/api/ConstrainULong.json)
* [`DoubleRange`](https://github.com/mdn/browser-compat-data/tree/master/api/DoubleRange.json)
* [`ULongRange`](https://github.com/mdn/browser-compat-data/tree/master/api/ULongRange.json)

Enums:

* [`RTCIceCandidateType`](https://github.com/mdn/browser-compat-data/tree/master/api/RTCIceCandidateType.json)
* [`RTCIceComponent`](https://github.com/mdn/browser-compat-data/tree/master/api/RTCIceComponent.json)

**Comment by foolip** [comment]
Note that some of the Constrain* types you listed are not dictionaries in [the spec](https://w3c.github.io/mediacapture-main/), just incorrectly claimed to be dictionaries on MDN. (They should still be removed IMHO.)

**Comment by foolip** [comment]
Another enum spotted in https://github.com/mdn/browser-compat-data/pull/10784: `XRSessionMode`

**Comment by foolip** [comment]
These are the *Init.json files we all, all dictionaries:

api/MutationObserverInit.json
api/RTCIceCandidateInit.json
api/VRLayerInit.json
api/DOMPointInit.json
api/RTCTrackEventInit.json
api/RTCRtpTransceiverInit.json

**Comment by queengooborg** [comment]
I've found some other dictionaries in our BCD by searching for the word "dictionary" in the MDN content, as well as `grep`ing the files for "options", "configuration", "parameters", etc.  There are...a lot of dictionaries still in BCD, it looks like.

I've updated the issue to turn it into a checklist of what we have left to remove and added all of the additional problematic files we've found.  I'm also pinning this issue since it's an important goal and even helps bring our real value percentage up some!

**Comment by Elchi3** [comment]
I believe https://developer.mozilla.org/en-US/docs/Web/API/Transferable is also a non-interface page / BCD data.

**Comment by queengooborg** [comment]
Good catch, thanks @Elchi3!  Yeah, this looks like something we can remove.  I've added it to the list!

**Comment by 1111elle1111** [comment]
#19293 #19259 #19261 #19265 #19270 found these what are they

**Comment by skyclouds2001** [comment]
> * RTCIceCredentialType

RTCIceCredentialType is removed via https://github.com/mdn/browser-compat-data/pull/18496 yet

**Comment by skyclouds2001** [comment]
> * RTCRtpSynchronizationSource

RTCRtpSynchronizationSource is also removed via https://github.com/mdn/browser-compat-data/pull/21759 yet

**Comment by skyclouds2001** [comment]
> * MediaTrackConstraints
> * MediaTrackSettings
> * MediaTrackSupportedConstraints

these three are in https://github.com/mdn/browser-compat-data/pull/21761

**Comment by Elchi3** [comment]
Thanks @skyclouds2001, I believe this is all merged and done now.

Closing 🎉
